services/121-service/src/registration/validators/is-not-both-empty.class.validator.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { validate } from 'class-validator';
+import { IsNotBothEmpty } from './is-not-both-empty.class.validator';
+
+class TestClass {
+  @IsNotBothEmpty<TestClass>('testOtherProperty')
```

Comment:
This is not a good idea because you will want to mark optional properties with @IsOptional() anyway, so your validator will be skipped, which can end in both properties being null/undefined.